### DISCUSSION TITLE
Yi Dome Guard

### DISCUSSION THREAD

**Discussion by guestisp** [roleoroleo/yi-hack-Allwinner-v2#627] [Q&A] [answered]
Guys, i'm looking for a PTZ camera working 100% in offline mode (no cloud access at all) and that i can integrate in HA in the near future.

The Yi Dome Guard sold on amazon italy seems a good product, but i don't know if this custom firmware is supported .
This is the camera: https://www.yitechnology.com/dome-R30/ this is the amazon link: https://www.amazon.it/YI-Videocamera-Sorveglianza-Rilevamento-Bidirezionale/dp/B09JVNW2M8 and i can see this model number `YSJR30GB_EU4`

Is this supported ?

Also, can I use an app to manage the cam without connecting it to a remote cloud ?

**Comment by guestisp** [comment]
no info? because i'm waiting for the purchase

**Comment by roleoroleo** [comment, marked as answer]
It is not possible to know for sure which model will be delivered after the purchase.
But the model number seems to be a r30gb.
Supported with some issues, read the readme.
